src/registrar/models/portfolio.py

```diff
@@ -21,11 +26,26 @@ class Portfolio(TimeStampedModel):
     creator = models.ForeignKey(
         "registrar.User",
         on_delete=models.PROTECT,
-        help_text="Associated user",
+        verbose_name="Portfolio creator",
         related_name="created_portfolios",
         unique=False,
     )
 
+    # Q for reviewers: shouldn't this be a required field?
```

Comment:
Maybe this would be skipped for Federal agency portfolios?

Reply:
This is auto filled for federal agencies with their federal agency name if federal, but is required on save for non federal (or at least should be). That way when displaying the name to call this portfolio we just display our desired org name as opposed to having to check if org is filled in and if not putting agency.